README.md

```diff
@@ -74,6 +74,7 @@ As the API has just launched, not many implementations exist yet. If you are int
 | [stevapple/swift-log-telegram](https://github.com/stevapple/swift-log-telegram) | a logging backend that sends log messages to any Telegram chat (Inspired by and forked from [wlisac/swift-log-slack](https://github.com/wlisac/swift-log-slack)) |
 | [jagreenwood/swift-log-datadog](https://github.com/jagreenwood/swift-log-datadog)  | a logging backend which sends log messages to the [Datadog](https://www.datadoghq.com/log-management/) log management service |
 | [google/SwiftLogFireCloud](https://github.com/google/swiftlogfirecloud)  | a logging backend for time series logging which pushes logs as flat files to Firebase Cloud Storage. |
+| [crspybits/swift-log-file](https://github.com/crspybits/swift-log-file.git)  | a simple local file logger |
```

Comment:
Mind if we add the implementation note: 

```suggestion
| [crspybits/swift-log-file](https://github.com/crspybits/swift-log-file)  | a simple local file logger (using `Foundation` `FileManager`) |
```

because there are some projects which want to / need to avoid depending on Foundation in the server side world.

// And changed the link to be to the repo, not git url itself.